File: help/pt_pt/exercicios_da_ficha_10/Ficha10_001/Ficha10_001.php
<?php
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    $i = 0 ;
    while(true ){
        echo  "Número positvo : " ;
        $i = intval(isset($_POST["i"])? $_POST["i"] : isset($_GET["i"]) ? $_GET["i"] : 0);
        
        echo $i . "<br/>";// print input 
        if($i > 0 ){
            break;
        }
        echo  "Erro na leitura\n\n" ;
    }
    echo  "Valor introduzido = " . $i  ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	quarta, 2022 out 12 19:26

?>

File: help/pt_pt/exercicios_da_ficha_10/Ficha10_001/Ficha10_001.c
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    int i = 0 ;
    while(1 ){
        printf("Número positvo : ");
        scanf("%d", &i);
        if(i > 0 ){
            break;
        }
        printf("Erro na leitura\n\n");
    }
    printf("Valor introduzido = %d",i );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 19:26
